inc/customizer/layouts.php

```diff
@@ -172,7 +172,9 @@ public function post_type_support() {
 		 *
 		 * @var array
 		 */
-		$post_types = (array) apply_filters( 'primer_layouts_post_types', get_post_types( array( 'public' => true ) ) );
+		$post_types = (array) apply_filters( 'primer_layouts_post_types', get_post_types( array(
+			'public' => true,
+		) ) );
```

Comment:
@EvanHerman I think it would be good to get into the habit of staying within the scope of the PR. When you least expect it a trivial fix will accidentally break something (I should know!) - then the change is harder to roll back because it was really just a hitchhiker on a commit in a branch that had a different purpose.

A suggestion would be to keep a branch hanging around locally, like `evans-tweaks`, where you can stash these type of trivial changes and to do each change in its own commit so it's really easy to revert if you ever need to.

Just my 2¢.